src/corehost/common/pal.unix.cpp

```diff
@@ -641,3 +641,9 @@ bool pal::is_running_in_wow64()
 {
     return false;
 }
+
+bool pal::are_paths_equal_with_normalized_casing(const string_t& path1, const string_t& path2)
+{
+    // On Unix paths are case-sensitive.
```

Comment:
The default Mac OS X file-system is case-insensitive.

Reply:
Thanks. New Commit to fix this